spec/soap_data/find_folder_legacy_request.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<soap:Envelope xmlns:soap="http://schemas.xmlsoap.org/soap/envelope/" xmlns:t="http://schemas.microsoft.com/exchange/services/2006/types" xmlns:m="http://schemas.microsoft.com/exchange/services/2006/messages">
+  <soap:Header>
+    <t:RequestServerVersion Version="Exchange2010_SP2"/>
+  </soap:Header>
+  <soap:Body>
+    <FindFolder xmlns="http://schemas.microsoft.com/exchange/services/2006/messages" Traversal="Deep">
```

Comment:
Just trying to learn a bit more about the problem, is this namespace invalid because it isn't `xmlns:t`? 🤔

Reply:
Yeah, exactly, that's the problem. The "t" vs "n" shortcuts are defined on the first line, in the "header".